packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/VirtualizedTraceView.tsx

```diff
@@ -150,6 +150,14 @@ function getCssClasses(currentViewRange: [number, number]) {
   });
 }
 
+export function getHexColorForSpan(span: Span) {
```

Comment:
a) what's the difference between hex color and rgb color? Can we use just one of these?
b) can we find a better place for this/these function(s)? components should not depend on each other, but you're importing from this module into other components